src/IPFSNode.ts

```diff
@@ -15,7 +15,7 @@ import PeerMonitor = require('ipfs-pubsub-peer-monitor');
 interface Message<T, P> {
   type: T;
   // Can be a store address or an ipfs peer id
-  to: string;
+  to?: string;
```

Comment:
I _think_ this is ok to do, to make a broadcasted message, but I'm not sure.

Reply:
Sure, it does, but this is for the case where a pinner joins the room when peers are already there.